src/pages/workspace/companyCards/assignCard/AssignCardFeedPage.tsx

```diff
@@ -77,6 +78,7 @@ function AssignCardFeedPage({route, policy}: AssignCardFeedPageProps) {
                 <ConfirmationStep
                     policyID={policyID}
                     backTo={backTo}
+                    workspaceMemberAccountID={workspaceMemberAccountID}
                 />
```

Comment:
I missed that we pass route params here. In that case we shouldn't use the current approach as we are still passing through `AssignCardFeedPage`. Instead we should store the first valid assignee email and compare it with the current one and if they are not the same let's not pass the backTo param. Something like this:

```js
    const firstAssigneeEmail = useRef(issueNewCard?.data.assigneeEmail);
    if (!firstAssigneeEmail.current) {
        firstAssigneeEmail.current = issueNewCard?.data.assigneeEmail;
    }
    const shouldUseBackToParam = !firstAssigneeEmail.current || firstAssigneeEmail.current === issueNewCard?.data.assigneeEmail;
```

Reply:
That code should go in `AssignCardFeedPage`